serving/samples/helloworld-kotlin/README.md

```diff
@@ -0,0 +1,213 @@
+                image: docker.io/{username}/helloworld-kotlin
+                env:
+                - name: TARGET
+                  value: "Kotlin Sample v1"
+    ```
+
+## Build and deploy this sample
+
+Once you have recreated the sample code files (or used the files in the sample
+folder) you're ready to build and deploy the sample app.
+
+1. Use Docker to build the sample code into a container. To build and push with
+   Docker Hub, run these commands replacing `{username}` with your
+   Docker Hub username:
+
+    ```shell
+    # Build the container on your local machine
+    docker build -t {username}/helloworld-kotlin .
+
+    # Push the container to docker registry
+    docker push {username}/helloworld-kotlin
+    ```
+
+2. After the build has completed and the container is pushed to docker hub, you
+   can deploy the app into your cluster. Ensure that the container image value
+   in `service.yaml` matches the container you built in
+   the previous step. Apply the configuration using `kubectl`:
+
+    ```shell
+    kubectl apply -f service.yaml
+    ```
+
+3. Now that your service is created, Knative will perform the following steps:
+   * Create a new immutable revision for this version of the app.
+   * Network programming to create a route, ingress, service, and load balance for your app.
+   * Automatically scale your pods up and down (including to zero active pods).
+
+4. To find the IP address for your service, use
+   `kubectl get svc knative-ingressgateway -n istio-system` to get the ingress IP for your
+   cluster. If your cluster is new, it may take sometime for the service to get assigned
+   an external IP address.
+
+    ```shell
+    kubectl get svc knative-ingressgateway -n istio-system
+    ```
+    ```shell
+    NAME                     TYPE           CLUSTER-IP     EXTERNAL-IP      PORT(S)                                      AGE
+    knative-ingressgateway   LoadBalancer   10.23.247.74   35.203.155.229   80:32380/TCP,443:32390/TCP,32400:32400/TCP   2d
+    ```
+
+5. To find the URL for your service, use
+    ```shell
+    kubectl get ksvc helloworld-kotlin  -o=custom-columns=NAME:.metadata.name,DOMAIN:.status.domain
+    ```
+    ```shell
+    NAME                DOMAIN
+    helloworld-kotlin   helloworld-kotlin.default.example.com
+    ```
+
+    > Note: `ksvc` is an alias for `services.serving.knative.dev`. If you have
+      an older version (version 0.1.0) of Knative installed, you'll need to use
+      the long name until you upgrade to version 0.1.1 or higher. See
+      [Checking Knative Installation Version](../../../install/check-install-version.md)
+      to learn how to see what version you have installed.
+
+6. Now you can make a request to your app to see the result. Replace `{IP_ADDRESS}`
+   with the address you see returned in the previous step.
+
+    ```shell
+    curl -H "Host: helloworld-kotlin.default.example.com" http://{IP_ADDRESS}
+    ```
+    ```shell
+    Hello World: Kotlin Sample v1
```

Comment:
The other changes made above mean this output should now be "Hello Kotlin Sample v1".